fractal/parser.py

```diff
@@ -720,6 +710,11 @@
 user_register_parser.add_argument(
     "new_password", help="Password for the new user"
 )
+user_register_parser.add_argument(
+    "--cache-dir",
+    help="---",  # FIXME
```

Comment:
Something like: "User's cache directory (necessary for workflow execution when using the SLURM backend)."